docs/docs/guide/debugging_table/table.tsx

```diff
@@ -0,0 +1,342 @@
+import React from 'react';
+import { detailsMap } from './details_map';
+
+export default function DebuggingInfoTable() {
+  const [details, setDetails] = React.useState('');
```

Comment:
Maybe we can split the state into separate variables, i.e. `tool`, `runtime`, `platform` etc.?